### PR TITLE
Form 341/select changes

### PR DESCRIPTION
**Pull request by matejpeluha**

### CODE REVIEW

File: next/public/locales/sk/forms.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
   },
   "selectAll": "Vybrať všetko",
   "deselectAll": "Odobrať všetko",
+  "options1": "Možnosti",
+  "options2": "Možností",
```
Comment (mpinter) [inline next/public/locales/sk/forms.json:11]: would use the _zero _one _few _many suffix (and don't forget to handle the grammar for 0 and 1)

https://www.i18next.com/translation-function/plurals

File: next/components/forms/widget-components/SelectField/Dropdown.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { EnumOptionsType } from '@rjsf/utils'
 import cx from 'classnames'
 import { FC } from 'react'
 
+import { useClickOutsideHandler } from '../../../utils/ClickOutsideHandler/useClickOutsideHandler'
```
Comment (mpinter) [inline next/components/forms/widget-components/SelectField/Dropdown.tsx:5]: why not the one from ts-hooks like here ? https://github.com/bratislava/bratislava.sk/pull/712/files

actually, would be nice to get rid of this custom one in the face of library one existing ,feel free to do it as part of this PR

**Review by mpinter — CHANGES_REQUESTED** [review]
couple minor changes

File: next/public/locales/sk/forms.json (hunk 1 above)
Comment (matejpeluha) [reply next/public/locales/sk/forms.json:11]: zero and one are not needded now but i can create these also

File: next/components/forms/widget-components/SelectField/Dropdown.tsx (hunk 2 above)
Comment (matejpeluha) [reply next/components/forms/widget-components/SelectField/Dropdown.tsx:5]: thanks for info, I thought that we use this in project. i will use one from ts-hooks

File: next/components/forms/widget-components/SelectField/SelectField.tsx [hunk 3]
```diff
@@ -65,6 +70,9 @@ const SelectFieldComponent: ForwardRefRenderFunction<HTMLDivElement, SelectField
   const [filter, setFilter] = useState<string>('')
   const [filterRef] = useState<RefObject<HTMLInputElement>>(React.createRef<HTMLInputElement>())
 
+  console.log('\nENUM OPTIONS', enumOptions)
+  console.log('VALUE:', value)
+
```
Comment (mpinter) [inline next/components/forms/widget-components/SelectField/SelectField.tsx]: probably forgotten

**Review by mpinter — APPROVED** [review]
🚢

**Pull request merged**